mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -354,6 +354,21 @@ var schemaNodePool = map[string]*schema.Schema{
 					ValidateFunc: validateIpCidrRange,
 					Description:  `The IP address range for pod IPs in this node pool. Only applicable if create_pod_range is true. Set to blank to have a range chosen with the default size. Set to /netmask (e.g. /14) to have a range chosen with a specific netmask. Set to a CIDR notation (e.g. 10.96.0.0/14) to pick a specific range to use.`,
 				},
+				"network_performance_config": {
+					Type:        schema.TypeList,
+					Optional:    true,
+					MaxItems:    1,
+					Description: `Network bandwidth tier configuration.`,
+					Elem:        &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"total_egress_bandwidth_tier": {
+								Type:        schema.TypeString,
+								Required:    true,
```

Comment:
Given this field is updatable, can you add a test updating from having it enabled to disabled?

Reply:
Ah, this is the first field under `network_config` that's updatable. GKE's complex update methods mean that updates are nonstandard for these resources- we'll need to update `nodePoolUpdate` to look for changes in `network_config` and send an update if they exist! That function's represented as a set of update call blocks, gated on specific field groups- any similar block using the `UpdateNodePoolRequest` type should serve as a reference.